docs/overview.md

```diff
@@ -19,34 +19,31 @@ Tremor is designed to handle the following:
 - Data Bridging - Tremor bridges asynchronous sources with synchronous sinks. Its ability to bridge sync/async and vice-versa enables message flows to be classified, dimensioned, segmented, and routed using user defined logic while handling back-pressure and other distributed systems problems on behalf of its operators.
 - Distribution at Scale - Tremor is designed for high volume messaging environments. Using proactive rate limiting and a novel algorithm to handle back-propagation, Tremor optimizes back-pressure handling at capacity up to 10x more efficiently than alternatives such as Logstash and Telegraf.
 - In-flight Redeployment - Tremor can be reconfigured via its API to allow workload migration and reconfiguration without redeployment.
-- Event processing - Tremor adopts principles from the Distributed Event Based Systems (DEBS), Event Stream Processor (ESP) and Complex Event Processing (CEP) communities; however, it currently encompasses an incomplete feature set. Over time, Tremor may evolve as an ESP or CEP solution, but this is not an explicit goal for the project.
+- Event processing - Tremor's primary goal is to operate as an Extract, Transform, and Loading (ETL) tool. While Tremor adopts principles from Distributed Event Based Systems (DEBS), Event Stream Processor (ESP) and Complex Event Processing (CEP) infrastructures, it is by no means an exhaustive system that encompasses all the aforementioned communities' features and requirements.
```

Comment:
This is a great synopsis of how @darach explained the Tremor architecture and functionality yesterday. But, I was thinking maybe we should maintain the element of possibility here. Like, "It's currently not exhaustive..." or, "It is by no means an exhaustive system... yet...". Something along those lines. What do you think?